src/components/markdown-styles.module.css

```diff
@@ -2,29 +2,90 @@
   @apply text-lg leading-relaxed;
 }
 
-.markdown p,
-.markdown ul,
-.markdown ol,
-.markdown blockquote {
+.markdown p {
   @apply my-6;
 }
 
+/* Links */
+.markdown a {
+  @apply text-primary;
+}
+.markdown a:hover {
+  @apply underline;
+}
+
+.markdown blockquote {
+  @apply p-4 mx-0 bg-accent-1 text-white mb-4 border-l-4 border-accent-3 italic;
```

Comment:
The color of the quotes seems to be too bright:

![image](https://user-images.githubusercontent.com/95120/109937079-e4489000-7cce-11eb-8d60-1bb9de11bbc5.png)